src/api/helpers/map-CIS-countries/map-CIS-country/map-NBI-issue-available/index.ts

```diff
@@ -8,7 +8,7 @@ const { CIS } = EXTERNAL_API_DEFINITIONS;
  * @param {String} NBI flag
  * @returns {Boolean}
  */
-export const mapNbiIssueAvailable = (str: string): boolean => {
+const mapNbiIssueAvailable = (str: string): boolean => {
   if (str === CIS.NBI_ISSUE_AVAILABLE.YES) {
```

Comment:
can add this to the above ticket to update this as well

Reply:
RE these comments, i'd rather leave as they are. The mapping logic will be changed, most likely throughout, in EMS-3842.